project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject ovotech/clj-gcp "0.4.1"
+(defproject ovotech/clj-gcp "0.4.2"
```

Comment:
```suggestion
(defproject ovotech/clj-gcp "0.5.0"
```
I guess you've technically broken any test implementations of `StorageClient` but since this isn't a "1.X.X" release semver either doesn't apply or suggests incrementing the minor version (I can't remember which)